orchestra/actions/util.py

```diff
@@ -12,20 +13,27 @@
 
 
 def run_script(script,
```

Comment:
I think this should be a `_run_script` to stress that you're not supposed to call it directly.